internal/app/ec2/ec2.go

```diff
@@ -51,9 +58,17 @@ func (e *EC2Fetcher) GetStatistics() Statistics {
 // GetDBInstanceTypeInformation returns information about specified AWS EC2 instance types
 // AWS RDS API use "db." prefix while AWS EC2 API don't so we must remove it to obtains instance type information
 func (e *EC2Fetcher) GetDBInstanceTypeInformation(instanceTypes []string) (Metrics, error) {
+	ctx, span := tracer.Start(e.ctx, "collect-ec2-metrics")
+	defer span.End()
```

Comment:
Not necessarily needed for this MR as a first iteration, but using a proxy Struct that only handles the definition of spans could avoid "polluting" the logic with tracing code.